deploy/ui/form.json

```diff
@@ -807,6 +807,38 @@
                   "description": "HTTPS hostname to set as redirect URL, when RH-SSO is configured"
                 }
               ]
+            },
+            {
+              "label": "Startup Strategy",
```

Comment:
In UI startup strategy should be set in common config part, not in compnents (for console or kie server)

Reply:
Ok